### ISSUE TITLE
Please add devices we need

### ISSUE THREAD

**Issue by freedomtan** [mlcommons/mobile_app_open#848]
- [x] mtk: @freedomtan 
- [x] samsung: @Mostelk 
- [x] Qualcomm: @mohitmundhragithub

**Comment by freedomtan** [comment]
Mediatek

|device name | SoC| notes|
|---:|----:|----|
|Xiaomi Redmi 10X 5G | Dimensity 820| 0.7|
|Vivo S9 5G | Dimensity 1100 | 1.0 |
|Oppo Find X5 Pro | Dimensity 9000 | 2.0| 
|Vivo X90 Pro | Dimensity 9200| 3.0|

**Comment by freedomtan** [comment]
Samsung from @Mostelk 


|device name | SoC| notes|
|---:|----:|----|
|Samsung Galaxy Note 20 ULTRA; SM-N985F| Exynos 990 |(v0.7)|
|Samsung Galaxy S21+ 5G SM-G996N |  Exynos 2100 |(v1.0)|
|Samsung Galaxy S22+ 5G SM-S906B | Exynos 2200 |(v2.0)|
|Samsung Galaxy S24+ 5G |  Exynos 2400 |  V4.0?|

**Comment by mohitmundhragithub** [comment]
<html>
<body>

device name | SoC | notes
-- | -- | --
Xiaomi 13 pro | SD 8Gen 2 | V3.0
Redmi Note 12 turbo | SD 7+ Gen 2 | V3.1
Asus ROG Phone 6 | SD 8+ Gen 1 | V2.1
Oppo Reno 8 pro (china variant) | SD 7 Gen 1 | V2.1
Xiaomi Mi 12 | SD 8 Gen 1 | V2.0
Xiaomi Mix 4 | SD 888+ 5G | V1.1



</body>
</html>

**Comment by Mostelk** [comment]
@mohitmundhragithub decision was to limit to 6 devices per org at any time, please chose accordingly, taking into account V4.0 submission.

**Comment by mohitmundhragithub** [comment]
> @mohitmundhragithub decision was to limit to 6 devices per org at any time, please chose accordingly, taking into account V4.0 submission.

sure. Removed the last 1.
We don't know the device to be used for V4.0 submission yet.

**Comment by freedomtan** [comment]
Samsung developer site has almost all the Samsung phones, https://developer.samsung.com/remotetestlab/docs/1/about-remote-test-lab/, https://developer.samsung.com/remote-test-lab

Can we do it automatically? I mean without integrated with our current github based CI/CD.

@anhappdev please help check it.

**Comment by anhappdev** [comment]
After some testing, here is what I found out about Samsung Remote Test Lab:
* We can either use the [web interface](https://developer.samsung.com/remotetestlab/devices) or [Android Studio](https://developer.samsung.com/sdp/blog/en/2021/06/04/using-remote-test-lab-with-android-studio#) to interact manually with a physical device (similar to how we interact with a simulator)
* I did not find a CLI program or an automated way to launch a device and run tests on it. Either using the web interface or Android Studio, we always need to open their website to start a session / launch a device.
* They have devices with the Exynos SoC. I can manually run the app with Samsung backend on the Samsung Galaxy S24 Ultra (Korea location).
